### PR TITLE
Add japanese translation of 0.21.2

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

File: _posts/ja/posts/2021-09-29-release-0.21.2.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+id: ja-blog-release-0.21.2
+lang: ja
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement.  English posts only
+announcement: 1
```
Comment (katesalazar) [inline _posts/ja/posts/2021-09-29-release-0.21.2.md:13]: Maybe this is supposed to be 0
"only true if announcement. **English posts only**"

Comment (azuchi) [reply _posts/ja/posts/2021-09-29-release-0.21.2.md]: Thanks. Fix at 451ad1a.

**Comment by harding** [comment]
Tested ACK 451ad1a58893c43b4351152d1ed449ddeeabebf9 Thanks!

**Comment by katesalazar** [comment]
@azuchi this could eventually suffer something like #844,
you might find use in adapting #846 here

**Comment by azuchi** [comment]
@katesalazar thanks. I fix it by #854.

**Pull request merged**